zaplib/web/wasm_runtime.ts

```
@@ -676,6 +676,10 @@ export const initialize: Initialize = (initParams) => {
         canvasData = initializeCanvas(canvas);
       }
 
+      rpc.receive(WorkerEvent.Panic, (e) => {
+        if (initParams.onPanic) initParams.onPanic(e);
```

@janpaul123 Thoughts on behavior if we do not specify a hook? Silently fail like this? I think I'd rather instead throw.

if (initParams.onPanic) { initParams.onPanic(e) } else { throw(e) }

---

Yeah I think throwing would be good. Maybe we can add a warning message in that case to let people know about the `onPanic` API? Something like "if you want to catch this error.." etc

---

Sure!

---

See last commit